helpers/api.js

```diff
@@ -0,0 +1,321 @@
+      {
+        Modul: "AXX",
+        Developer: [
+          {
+            DevloperId: "4",
+            Firstname: "Andrea",
+            Lastanme: "See",
+          },
+          {
+            DeveloperId: "3",
+            Firstname: "Simone",
+            Lastanme: "Winterberg",
+          },
+        ],
+        BPA: [
+          {
+            BpaId: "4",
+            Firstname: "Thorsten",
+            Lastanme: "Kurz",
+          },
+        ],
+      },
+    ],
+  },
+  {
+    Id: "3",
+    Solution: "Verkauf",
+    Team: "AA",
+    BPE: "Heidrung Maier",
+    BSEINT: "Andrea Meier",
+    BSEGR: "Diana Carosi",
+    LeadDevloper: "Luka Ambrosios",
+    CBO: "Serkan Turhan",
+    SupportGroup: "supp-verkauf",
+    Modules: [
+      {
+        Modul: "DKP",
+        Developers: [
+          {
+            DeveloperId: "2",
+            Firstname: "Jarno",
+            Lastanme: "Recklinghausen",
+          },
+        ],
+        BPA: [
+          {
+            BpaId: "2",
+            Firstname: "Carmen",
+            Lastanme: "Müller",
+          },
+        ],
+      },
+      {
+        Modul: "WIB",
+        Developer: [
+          {
+            DeveloperId: "3",
+            Firstname: "Simone",
+            Lastanme: "Winterberg",
+          },
+        ],
+        BPA: [
+          {
+            BpaId: "3",
+            Firstname: "Laura",
+            Lastanme: "Hammer",
+          },
+        ],
+      },
+      {
+        Modul: "XML",
+        Developer: [
+          {
+            DevloperId: "4",
+            Firstname: "Andrea",
+            Lastanme: "See",
+          },
+          {
+            DeveloperId: "3",
+            Firstname: "Simone",
+            Lastanme: "Winterberg",
+          },
+        ],
+        BPA: [
+          {
+            BpaId: "3",
+            Firstname: "Laura",
+            Lastanme: "Hammer",
+          },
+          {
+            BpaId: "4",
+            Firstname: "Thorsten",
+            Lastanme: "Kurz",
+          },
+        ],
+      },
+    ],
+  },
+  {
+    Id: "4",
+    Solution: "Verwaltung",
+    Team: "AA",
+    BPE: "Sebastian Brucker",
+    BSEINT: "Reiko Carlsen",
+    BSEGR: "Diana Carosi",
+    LeadDevloper: "Sergej Seehl",
+    CBO: "Thomas Bauernfeind",
+    SupportGroup: "supp-verwaltung",
+    Modules: [
+      {
+        Modul: "NHC",
+        Developers: [
+          {
+            DeveloperId: "3",
+            Firstname: "Simone",
+            Lastanme: "Winterberg",
+          },
+        ],
+        BPA: [
+          {
+            BpaId: "3",
+            Firstname: "Laura",
+            Lastanme: "Hammer",
+          },
+        ],
+      },
+    ],
+  },
+];
+
+function getAllData() {
+  return data;
+}
+
+function getDataById(id) {
+  return data.find((solution) => solution.id === id);
+}
+
+const hostname = "http://localhost:3000/";
+
+function fetcher(url) {
+  if (!url) return;
+
+  return fetch(hostname + url).then((res) => res.json());
+}
```

Comment:
this code doesn't seem to be needed here. as such it should not be part of this PR.